src/m2n/M2N.hpp

```diff
@@ -182,6 +191,15 @@ class M2N
   bool _isMasterConnected = false;
 
   bool _areSlavesConnected = false;
+
+
+  /// between two serial participants, only use the master-master com (required only in some unit tests)
+  bool _useOnlyMasterCom = false;
```

Comment:
Can you explain the rational behind this?

Reply:
Rational is explained in the PR description:

> This makes live a bit more complicated for unit tests in cplscheme as they need inter-participant communication. I decided to not construct the complete overhead of mesh and vertex distribution definition for every test, but offered a loophole through a friend declaration in M2N and a fallback to the pure master-master communication. Suggestions welcome!

Reply:
I'll copy this to the code.

Reply:
Done